### ISSUE TITLE
Improving the  HyperParameter Tuning part with  Keras-tuner  for CNN Model

### ISSUE THREAD

**Issue by akramIOT** [CiscoAI/deep-learning-wireless-localization#13]
As per my  run of this code, I see this  Adam optimizer for  CNN  specifically for this  dataset  there  is only  certain level of  accuracy  which can be achieved.  Can I submit a PR with  Keras-tuner  based  CNN model  with good accuracy  score for this  same  BLE_RSSI_ location  dataset   ?  .  Thanks
